test/libsolidity/syntaxTests/enums/literal_conversion_error.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+    enum Test { One, Two }
+    function f() public {
+        Test(-1);
+        Test(2);
+        Test(13);
+        Test(5/3);
+    }
```

Comment:
I'd add some positive tests:
- That 0 and the highest enum value do work.
- Do we have a test that for non-literals the highest enum value works and hightest value + 1 does not?